crud-routing/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "crud-routing",
+  "version": "1.0.0",
+  "description": "CRUD Menu items",
+  "main": "index.js",
+  "scripts": {
+    "start": "index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.20.2",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "nodemon": "^2.0.22"
```

Comment:
nodemon should be in dev dependency

Reply:
changed nodemon dependencies to dev dependency